subjects/road_intersection/audit/README.md

```diff
@@ -1,86 +1,98 @@
+##### Try to generate two vehicles at the same time, one using the `"Right"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Up"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Left"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (to this at least 3 times).
+##### Try to generate two vehicles at the same time, one using the `"Down"` key and the other using the `"Right"` key (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the key `"Right"`.
+##### Try to generate five vehicles using the `"Up"` key, at the same time generate two vehicles using the `"Right"` key.
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
-##### Try to generate one vehicle for all the lanes (to this at least 3 times).
+##### Try to generate one vehicle for all the lanes (do this at least 3 times).
 
-###### Did all vehicles passed the intersection without any collision?
+###### Did all the vehicles passed the intersection without any collision?
 
 ##### Try to generate vehicles randomly using the `"R"` key. Then wait for at least 1 min.
```

Comment:
This seems to be a less-specific duplicate of another question. It does have a 1 minute wait though, what does that test?

https://github.com/01-edu/public/blob/332e922aaec357b2a8b5e9ecc25c8f929de95701/subjects/road_intersection/audit/README.md?plain=1#L27

Reply:
one is to confirm if the R feature is working and the cars are created at a safe distance, and the other is to check if there is any crash when generating vehicles with the R